staging/src/k8s.io/apiserver/pkg/storage/etcd3/healthcheck.go

```diff
@@ -34,7 +34,7 @@ func EtcdHealthCheck(data []byte) error {
 		return err
 	}
 	if obj.Health != "true" {
-		return fmt.Errorf("Unhealthy status: %s", obj.Health)
+		return fmt.Errorf("unhealthy status: %s", obj.Health)
```

Comment:
Let's not change this even though it might be flagged. Someone might be using this error text and we don't want to break them by changing the case here.

Reply:
Resolved.